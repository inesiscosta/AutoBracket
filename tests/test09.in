50 75
42 1 25 27 43 24 32 33 46 41 48 49 50 15 40 39 33 47 42 24 8 44 46 18 48 32 41 37 41 12 30 36 36 40 5 45 8 41 6 46 32 32 13 48 17 1 33 22 33 45
25 6 44 33 25 39 42 17 25 24 35 47 26 50 12 41 17 12 14 14 30 1 47 48 35 8 49 23 42 14 37 5 15 45 13 47 32 13 13 43 47 21 11 20 47 19 30 42 26 44
31 26 47 25 5 4 45 8 34 13 37 12 27 48 37 9 38 4 29 41 9 39 37 20 46 1 48 2 9 5 50 45 28 1 27 41 33 38 1 8 6 49 23 7 47 17 28 45 31 29
24 49 46 35 26 24 15 41 18 44 23 33 7 40 1 2 5 23 2 9 34 1 7 34 17 27 1 46 2 44 3 27 20 24 14 35 8 28 47 22 1 42 27 9 9 37 36 25 47 50
47 27 50 31 26 14 16 22 9 27 7 48 25 18 8 37 42 17 16 26 42 9 35 19 28 31 43 45 1 2 8 32 32 24 1 25 26 46 45 9 23 8 11 25 34 18 1 17 20 49
48 18 30 41 14 22 4 4 12 30 26 27 43 25 29 13 30 18 29 34 8 43 3 46 2 24 24 25 22 10 17 28 39 4 3 11 29 19 12 26 1 50 31 28 28 44 2 13 14 44
5 34 50 27 16 31 31 27 8 4 46 36 10 46 35 43 35 9 18 46 38 10 25 4 18 5 49 50 32 29 49 42 21 37 15 43 34 16 44 14 44 46 38 17 12 3 18 34 45 32
44 13 46 14 1 9 4 26 44 18 27 14 47 31 9 4 18 33 35 10 41 23 15 15 43 37 1 43 32 19 21 31 18 9 15 13 28 4 40 16 36 25 5 3 25 15 27 14 42 2
19 15 45 35 27 20 49 39 12 10 14 4 21 7 12 11 36 32 40 22 4 44 38 13 10 46 27 2 12 25 50 18 18 36 15 32 20 11 43 31 45 5 42 7 9 24 8 49 26 35
41 30 16 1 45 49 1 35 21 38 47 18 29 41 40 25 44 4 41 28 18 9 15 43 9 34 13 8 34 40 25 8 34 12 38 49 35 24 3 46 12 46 14 27 45 2 45 2 43 17
8 43 30 46 26 37 3 14 24 44 1 26 30 18 29 25 7 15 2 7 33 42 43 15 9 30 6 13 17 7 49 35 38 41 42 2 15 19 50 17 40 45 33 30 35 46 46 19 24 46
38 15 29 20 28 8 3 44 45 21 37 13 24 49 24 45 38 2 31 2 13 48 20 29 45 2 5 49 13 30 28 3 42 19 44 32 32 9 9 49 38 44 30 27 12 6 30 27 48 19
17 24 48 9 23 39 14 29 42 42 14 39 8 28 26 40 32 25 38 10 26 8 46 13 9 20 47 3 44 19 8 20 27 10 42 33 15 28 41 37 29 39 24 40 13 31 16 12 36 33
34 23 39 43 25 3 1 30 17 1 47 28 46 17 8 47 32 37 30 1 45 12 34 8 14 22 39 18 41 2 15 4 6 24 35 33 10 14 29 48 32 45 23 7 27 22 13 21 34 37
38 16 43 28 9 26 29 25 19 31 18 31 23 38 26 34 37 30 33 13 19 9 13 21 42 3 27 44 35 25 45 23 34 47 45 49 29 32 36 2 38 6 11 24 7 48 17 32 33 5
22 38 11 22 39 38 14 36 46 50 44 25 37 6 10 24 18 14 44 38 30 27 15 36 48 36 21 26 16 11 16 1 19 22 39 5 19 27 41 22 2 14 26 6 40 19 21 15 38 27
35 22 11 23 10 26 35 35 48 15 45 30 7 9 28 5 23 22 46 24 10 13 14 14 1 7 35 8 25 5 50 21 4 48 39 45 48 46 33 23 7 46 33 33 26 41 1 20 32 26
27 42 26 17 3 35 13 23 16 5 45 38 18 9 39 4 34 46 39 37 6 23 10 9 7 27 6 32 25 7 35 49 11 7 17 18 50 43 50 9 24 17 44 27 28 15 41 24 34 31
24 29 10 36 37 12 5 37 37 27 44 30 16 48 25 9 30 41 48 20 9 34 16 4 36 36 18 1 50 36 28 27 10 22 12 12 49 44 20 30 49 49 35 21 26 46 14 44 8 5
13 7 46 16 27 25 16 1 4 23 39 4 38 40 36 19 33 44 45 43 3 14 27 49 39 2 22 19 1 4 28 10 3 33 18 39 15 7 46 22 23 14 46 9 24 45 32 49 46 45
10 9 38 47 48 47 30 48 23 4 44 40 4 43 33 36 48 27 14 9 38 25 25 7 1 25 21 24 28 15 11 11 29 36 33 29 39 2 20 23 3 32 12 7 9 15 38 15 19 48
5 12 43 21 32 9 16 29 10 14 8 15 8 36 17 43 1 25 4 5 34 1 20 15 2 44 22 26 8 8 45 43 19 30 11 36 7 36 21 1 36 31 23 50 32 21 46 49 36 43
42 49 16 17 21 48 46 2 7 49 31 6 43 31 47 16 23 44 43 4 21 9 17 44 45 14 32 39 42 19 32 32 6 46 31 27 28 33 28 18 19 9 2 24 16 9 29 33 42 2
1 47 20 25 7 32 29 10 31 48 42 46 38 8 37 29 2 37 42 29 32 30 3 8 49 11 21 48 38 25 17 26 39 8 17 49 36 3 11 50 43 6 30 24 12 17 22 7 43 11
49 50 44 35 2 38 5 25 9 24 16 39 14 8 14 24 24 13 12 30 9 34 10 7 31 22 33 26 32 44 27 24 47 49 35 15 24 31 6 28 43 46 17 16 5 23 22 48 23 44
29 41 22 14 33 49 40 16 11 25 28 41 16 21 5 29 12 4 38 12 19 16 3 43 50 14 9 34 27 20 2 44 41 31 46 47 39 5 43 16 3 6 34 11 50 41 33 22 1 6
41 40 4 45 49 50 48 4 42 10 10 50 36 18 42 48 13 3 14 2 10 35 48 29 17 6 45 2 49 43 50 31 37 9 19 32 36 5 10 30 27 49 42 38 32 15 17 47 4 28
9 40 39 20 50 29 3 15 9 2 13 33 17 11 12 44 20 8 2 45 22 3 15 40 7 12 22 48 20 23 17 49 7 23 41 8 27 40 21 39 10 31 22 31 37 45 12 23 28 14
18 36 37 33 24 39 17 42 19 34 33 25 39 5 17 22 10 4 25 16 32 24 43 18 43 1 27 36 27 32 44 12 29 4 18 9 39 16 41 47 38 31 47 34 7 16 40 1 18 38
3 25 25 46 46 27 34 30 50 32 35 26 6 33 33 50 44 10 37 11 3 34 39 40 3 26 9 16 1 48 12 23 31 33 49 35 26 25 11 17 39 25 27 20 8 44 42 45 44 35
3 4 48 28 37 19 38 34 39 35 43 36 11 27 11 13 41 38 39 4 33 47 20 50 28 43 12 46 40 13 38 27 7 9 19 38 41 11 14 49 23 40 49 35 46 14 18 9 32 20
34 24 37 1 10 34 15 23 43 3 7 47 39 44 22 43 34 24 11 39 7 50 12 5 37 23 37 3 50 47 18 38 31 3 22 18 40 48 2 45 9 13 45 39 12 7 15 37 17 14
5 1 39 33 31 9 6 32 33 40 48 37 8 17 24 30 3 28 12 47 50 11 38 6 2 19 9 43 20 34 48 42 30 5 32 32 32 43 11 3 5 19 33 46 43 30 22 23 3 37
2 1 37 10 32 43 21 12 11 46 45 41 9 30 31 14 2 22 5 16 17 2 32 29 6 12 2 17 41 34 6 34 46 27 44 18 41 32 37 40 47 43 45 49 29 13 18 24 45 23
19 40 34 41 29 33 18 13 14 20 28 10 22 6 5 36 13 32 26 47 4 46 7 40 2 42 10 9 23 40 45 34 7 34 34 7 27 5 12 41 27 10 38 50 33 18 42 26 28 39
37 9 42 12 39 34 18 10 38 12 39 23 17 39 41 4 27 47 43 37 31 8 6 11 7 14 35 19 40 49 22 3 5 48 34 44 37 3 6 11 20 27 30 31 23 43 26 31 47 32
40 4 2 16 38 34 22 45 8 50 15 47 17 14 33 6 45 32 44 50 37 6 23 5 16 17 5 50 10 47 38 20 31 12 50 6 19 19 35 17 27 41 40 42 46 30 31 13 40 17
40 50 34 38 28 17 20 13 25 40 8 25 20 47 29 45 12 5 10 8 48 45 18 19 20 18 19 24 4 5 33 22 4 18 39 8 7 30 9 49 30 10 17 50 16 25 8 9 22 10
50 46 17 9 25 18 38 12 11 32 25 2 12 12 19 50 1 45 14 11 39 19 41 2 10 20 24 28 16 25 8 14 3 22 24 6 1 47 46 20 19 10 10 19 45 13 22 12 20 33
21 3 29 14 11 33 37 44 12 44 20 3 4 5 44 47 46 6 50 2 36 47 24 42 8 27 5 34 2 17 10 18 43 5 1 17 24 24 37 46 18 10 50 47 12 38 9 24 12 31
11 6 45 17 10 31 43 14 26 10 32 2 33 39 8 20 2 38 1 17 36 20 5 47 17 48 22 14 12 27 34 20 32 44 5 16 47 38 33 8 18 19 13 31 50 14 43 39 5 48
28 37 8 19 2 10 8 35 22 19 14 44 7 29 43 46 18 5 19 7 36 38 27 48 38 21 23 21 40 49 11 24 26 25 27 47 30 28 24 34 29 34 34 5 24 16 11 29 45 44
34 20 15 14 35 22 18 31 2 11 18 16 34 33 34 13 25 38 23 6 33 7 33 26 18 31 37 47 5 47 17 36 16 4 36 35 46 16 34 34 49 19 8 10 44 43 25 3 18 45
40 50 30 15 48 13 10 25 4 12 29 35 24 15 3 41 11 30 46 47 15 40 1 1 4 25 22 44 40 4 33 15 18 17 41 8 36 15 49 43 36 42 36 23 33 50 39 23 22 27
22 30 20 27 20 2 13 2 35 25 21 33 44 50 38 34 33 50 16 19 37 21 48 29 49 38 41 22 35 20 8 32 3 34 7 47 9 13 6 18 13 19 8 7 48 28 16 48 39 7
48 19 41 33 28 34 7 45 27 45 9 27 8 29 23 50 5 2 48 35 12 15 6 14 23 48 15 18 18 19 9 21 38 24 48 25 12 44 30 1 20 34 1 29 27 43 30 24 35 41
44 38 17 10 28 14 42 48 32 12 6 30 27 25 28 29 15 8 4 46 16 32 35 19 36 38 23 36 25 13 49 40 40 19 8 48 1 24 42 2 31 38 2 37 9 41 4 31 14 6
40 38 20 22 7 44 41 17 27 6 24 49 4 46 24 36 5 1 10 12 6 8 10 39 42 13 38 50 37 39 33 6 48 38 44 35 33 23 35 48 41 40 10 15 16 16 39 23 3 42
25 45 32 18 37 30 40 12 28 22 44 38 45 11 4 15 49 7 32 26 27 39 6 35 7 24 7 30 14 14 32 3 14 30 14 39 33 21 12 28 22 44 4 43 23 31 1 38 22 1
35 1 44 16 5 18 12 27 36 27 46 36 26 1 13 35 19 4 8 38 12 43 16 21 32 32 48 38 15 26 28 27 5 7 34 14 31 22 48 1 34 5 25 44 45 42 35 14 13 49
40 17 36 13 26 10 5 19 4 45 34 5 19 45 29 30 3 10 17 38 47 14 5 5 12 12 14 49 30 31 8 27 46 30 49 47 41 31 25 47 37 22 1 24 15 27 38 7 17 15 43 12 30 50 3 29 27 27 50 39 15 43 43 5 12 16 10 1 1 18 20 47 40 44 34
17
