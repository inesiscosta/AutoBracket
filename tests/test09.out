1
((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((40 17) 36) 13) 26) 10) 5) 19) (4 (45 34))) (5 19)) 45) (29 30)) 3) 10) 17) 38) 47) 14) ((5 5) 12)) 12) 14) 49) 30) 31) 8) 27) 46) 30) 49) 47) 41) 31) 25) 47) 37) 22) 1) 24) 15) 27) 38) 7) 17) 15) 43) 12) 30) 50) 3) 29) 27) 27) 50) 39) 15) 43) 43) 5) 12) 16) (10 1)) 1) 18) (20 47)) 40) 44) 34)
